modyn/tests/supervisor/internal/eval_strategies/test_offset_eval_strategy.py

```diff
@@ -0,0 +1,37 @@
+from modyn.supervisor.internal.eval_strategies import OffsetEvalStrategy
+
+
+def test_get_eval_interval():
+    config = {"offsets": ["-inf", "inf", "0s", "100s", "-100s"]}
+    eval_strategy = OffsetEvalStrategy(config)
+    assert list(eval_strategy.get_eval_interval(0, 0)) == [
+        (0, 0),
+        (1, None),
+        (0, 1),
+        (1, 101),
```

Comment:
I don't understand the +1 logic here (I have seen the comment in the implementation). So if you offset 100s from 0, the eval, why is it not (0,100)?

Reply:
because the samples with timestamp `0` is *included* in this trigger. So when we produce an interval on future data, we should *exclude* the samples with timestamp `0`. That's why we add 1.

Reply:
I also modified the comments to hopefully make it clearer.

Reply:
Ahh, I see. I guess with the future perspective it makes sense :) Thank you!